examples/animation/test.js

```diff
@@ -0,0 +1,114 @@
+var chai = require("chai");
+const { MaterialHandler } = require("../../build/playcanvas");
+const { replace } = require("sinon");
+
+function describe(name, func) {
+	console.log(name);
+	func()
+}
+
+function it(name, func) {
+	//console.log(name);
+	try {
+		func();
+	} catch (e) {
+		console.log(name, "failed: ", e)
+	}
+}
+
+global.describe = describe;
+global.it = it;
+
+global.expect = chai.expect;
+
+global.strictEqual = function (a, b) {
+    expect(a).to.equal(b);
+};
+
+global.equal = function (a, b) {
+    expect(a).to.equal(b);
+}
+
+global.notEqual = function (a, b) {
+    expect(a).to.not.equal(b);
+}
+
+global.deepEqual = function (a, b) {
+    expect(a).to.deep.equal(b);
+}
+
+global.ok = function (a) {
+    expect(a).to.exist;
+}
+
+global.close = function (a, b, e) {
+    expect(a).to.be.closeTo(b, e);
+}
+
+
+fetch = require("file-fetch");
+fs = require("fs");
+
+global.window = {
+	navigator: {
+		userAgent: "node"
+	}
+};
+
+require("./Loader");
+//console.log("Loader", Loader)
+
+async function load_wasm() {
+	var wasmFile = fs.readFileSync("../../build/untouched.wasm");
```

Comment:
I guess it's just temporary for debugging? Otherwise it could be
```suggestion
	var wasmFile = fs.readFileSync("../../build/optimized.wasm");
```

Reply:
Thanks! It's a script for executing the unit tests, so eventually it should test both versions

Reply:
I see